lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -2433,8 +2432,22 @@ def _GetDefaultMachineVersion(cls, kvm_path):
     else:
       return "pc"
 
-  @classmethod
-  def _StopInstance(cls, instance, force=False, name=None, timeout=None):
+  def _WaitForInstanceToShutdown(self, timeout, name):
+    """Wait for QEMU to shut down after timeout or kill it
```

Comment:
It seems nothing is killed here, so the comment might be to much?

Reply:
oh, that is a leftover comment from an earlier version of the method. I'll fix that and merge the PR.

Thanks for spotting this! :-)

Reply:
Fixed